common/config/config.go

```diff
@@ -2,23 +2,34 @@ package config
 
 import (
 	"github.com/HackIllinois/api/common/configloader"
+	"github.com/HackIllinois/api/common/errors"
 	"os"
+	"strings"
 )
 
 var IS_PRODUCTION bool
+var DEBUG_MODE bool
 
 func init() {
 	cfg_loader, err := configloader.Load(os.Getenv("HI_CONFIG"))
 
 	if err != nil {
-		panic(err)
+		panic(errors.InternalError(err.Error(), "Could not load from config file."))
 	}
 
 	production, err := cfg_loader.Get("IS_PRODUCTION")
 
 	if err != nil {
-		panic(err)
+		panic(errors.InternalError(err.Error(), "Could not get variable IS_PRODUCTION from configloader."))
 	}
 
-	IS_PRODUCTION = (production == "true")
+	IS_PRODUCTION = (strings.ToLower(production) == "true")
```

Comment:
Let's just do `== "true"` to enforce a single possible formatting of true in the config file.